### PR TITLE
GC Reinitialize Sweep For Restore (CRIU) (0.38)

### PR DESCRIPTION
**Pull request by RSalman**
Cherry-pick of https://github.com/eclipse/omr/pull/6892

### CODE REVIEW

**Comment by dmitripivkine** [comment]
@pshipton FYI This is double delivery for 0.38 for https://github.com/eclipse/omr/pull/6892

**Comment by pshipton** [comment]
We can wait until tomorrow after the nightly build to merge this since https://github.com/eclipse/omr/pull/6892 was only promoted this morning.

**Review by pshipton — APPROVED** [review]

**Pull request merged**
